### PR TITLE
MF-1128 - Add golangci-linter to a CI script

### LINKED ISSUE #1128: Add golangci-linter to a CI script
**FEATURE REQUEST**

1. Is there an open issue addressing this request? If it does, please add a "+1" reaction to the
   existing issue, otherwise proceed to step 2.

2. Describe the feature you are requesting, as well as the possible use case(s) for it.
We have the same problem as those guys: https://github.com/square/go-jose/issues/298
[golangci-lint](https://github.com/golangci/golangci-lint) should be added to out CI script.

3. Indicate the importance of this feature to you (must-have, should-have, nice-to-have).
Must-have

### PR DESCRIPTION
**Pull request by blokovi**
Resolves #1128

### CODE REVIEW

**Review by manuio — APPROVED** [review]
great! LGTM

**Review by mteodor — APPROVED** [review]

File: sdk/go/channels_test.go [hunk 1]
```diff
@@ -141,7 +141,7 @@ func TestCreateChannels(t *testing.T) {
 		res, err := mainfluxSDK.CreateChannels(tc.channels, tc.token)
```
Comment (drasko) [inline sdk/go/channels_test.go:141]: Why changing these other files? Is it lint that is complaining?

**Review by drasko — CHANGES_REQUESTED** [review]

File: sdk/go/channels_test.go (hunk 1 above)
Comment (drasko) [reply sdk/go/channels_test.go:141]: Looks like yes, so I will approve.

**Review by drasko — APPROVED** [review]
LGTM

File: sdk/go/channels_test.go (hunk 1 above)
Comment (blokovi) [reply sdk/go/channels_test.go:141]: yes

**Pull request merged**